### PR TITLE
Add author information for Njagi Mwaniki

### PR DESCRIPTION
**Pull request by urbanslug**
Add author information for Njagi Mwaniki, affiliation and orcid.

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -51,8 +51,8 @@ authors:
     orcid: 0000-0000-0000-0000
     affiliation: 
   - name: Njagi Mwaniki
-    orcid: 0000-0000-0000-0000
-    affiliation: 
+    orcid: 0000-0002-4858-2375
+    affiliation: Pwani University
```
Comment (subwaystation) [inline paper/paper.md]: Please add your affiliation in lines 68 and following. Here, you can just add the number of the affiliation. Thanks!

Comment (subwaystation) [reply paper/paper.md]: @urbanslug

Comment (urbanslug) [reply paper/paper.md]: Done in https://github.com/graph-genome/graph-genome.github.io/pull/24/commits/6c68101c24e546c4aad2d7ca9d3a1960de66bef6

**Comment by subwaystation** [comment]
Thank @urbanslug

**Pull request merged**